src/main/java/lotto/model/LottoNumber.java

```diff
@@ -1,25 +1,30 @@
 package lotto.model;
 
-import java.util.Objects;
 import lotto.exception.LottoNumberException;
 
+import java.util.Objects;
+
 public final class LottoNumber {
 
-  private static final int MIN_VALUE = 1;
-  private static final int MAX_VALUE = 45;
+  static final int MIN_VALUE = 1;
+  static final int MAX_VALUE = 45;
   private final int value;
```

Comment:
상수와 인스턴스 변수의 구분을 위해서 공백 라인을 하나 추가해보면 어떨까 싶습니다.
```suggestion
  static final int MAX_VALUE = 45;
  
  private final int value;
```